src/mme-app/conf/mme.json

```diff
@@ -19,7 +19,11 @@
 			"dig1": 0,
 			"dig2": 1,
 			"dig3": -1
-		}
+		},
+        "plmnlist": { 
+			"plmn1": "mcc=315,mnc=010",
+			"plmn2": "mcc=208,mnc=01"
+        }
```

Comment:
Which one you use for your Mme testing ?

Reply:
We are using 208 01.